deployment-basics/step4/verify.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+  
+IMAGE=$(kubectl get deployment my-first-deployment -o json | jq .spec.template.spec.containers[0].image)
+
+if [ "$IMAGE" = "\"httpd:alpine\"" ]; then
```

Comment:
Might also be possible to do just:

`kubectl get deployment my-first-deployment -o json | jq .spec.template.spec.containers[0].image | grep httpd:alpine`